src/System.Security.Cryptography.X509Certificates/tests/TestEnvironmentConfiguration.Unix.cs

```diff
@@ -0,0 +1,85 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+
+namespace System.Security.Cryptography.X509Certificates.Tests
+{
+    internal static partial class TestEnvironmentConfiguration
+    {
+        static partial void DetermineCanModifyStores(ref bool canModify)
```

Comment:
I can't figure out why you use ref rather than just a return value, and why you have two functions, one with param and one without?